dist/docker/debian/build_docker.sh

```diff
@@ -69,7 +69,7 @@ packages=(
     "build/dist/$config/debian/$product-conf_$version-$release-1_$arch.deb"
     "build/dist/$config/debian/$product-kernel-conf_$version-$release-1_$arch.deb"
     "build/dist/$config/debian/$product-node-exporter_$version-$release-1_$arch.deb"
-    "tools/cqlsh/build/debian/$product-cqlsh_$version-$release-1_all.deb"
```

Comment:
@yaronkaikov 

Why reruns of the CI, doesn't pick this change ? and keep failing to fetch the wrong cqlsh file ?

Is this file cached somewhere ?
I don't get it

Reply:
My guess is that the submodule part is been done from `scylladb/scylladb` and not from your fork

You should probably go to your branch into the `tools/cqlsh` and checkout your changes and then add a commit

Reply:
but that specific code is in scylla core.

and my cqlsh submodule is pointed to my fork.